index.js

```diff
@@ -172,9 +221,15 @@ client.manager = new Manager({
     const response = await client.manager.search(serverQueue.songs[0].url);
     player.play(response.tracks[0]);
   })
-  .on("socketClosed", async (player) => {
```

Comment:
Is this event still used anywhere as it has been removed?

Reply:
@Darkflame72 That bit of code was removed from the file. I'm not sure why it's showing here but if you go to the branch it isn't there

Reply:
No it isn't. It has been replaced by playerMoved, which is emitted on disconnect and move to another channel